### PR TITLE
removes secborgs

### PR DESCRIPTION
**Pull request by ParadoxSpace**
The evil will finally be defeated.
This takes security borgs out of the game entirely. They bring pretty much nothing to the table, in terms of either roleplay or positive influence to the station.
Also, combat and syndicate modules now have cuffs.

### CODE REVIEW

**Comment by ParadoxSpace** [comment]
!review

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 1]
```diff
@@ -523,52 +522,6 @@ var/global/list/robot_modules = list(
...
-				)
-
-/obj/item/weapon/robot_module/security/general/New()
-	..()
-	src.modules += new /obj/item/weapon/crowbar(src)
-	src.modules += new /obj/item/device/flash(src)
-	src.modules += new /obj/item/borg/sight/hud/sec(src)
-	src.modules += new /obj/item/weapon/handcuffs/cyborg(src)
-	src.modules += new /obj/item/weapon/melee/baton/robot(src)
-	src.modules += new /obj/item/weapon/gun/energy/taser/mounted/cyborg(src)
```
Comment (Karolis2011) [inline code/modules/mob/living/silicon/robot/robot_modules.dm]: I am pretty sure this makes taser and tasercooler upgrade unused. It should be removed too.

Comment (Sindorman) [reply code/modules/mob/living/silicon/robot/robot_modules.dm]: No, your BST borg uses it

Comment (Karolis2011) [reply code/modules/mob/living/silicon/robot/robot_modules.dm]: Then upgrade is unused.

Comment (Sindorman) [reply code/modules/mob/living/silicon/robot/robot_modules.dm]: True. Remove the upgrade. Keep the taser itself

**Comment by lolman360** [comment]
how will i get my sillycone balids now, mister spaceparadoxed?

based and redpilled pr

**Comment by ParadoxSpace** [comment]
https://github.com/Aurorastation/Aurora.3/pull/6733

File: code/modules/mob/living/silicon/robot/robot_modules.dm (hunk 1 above)
Comment (Karolis2011) [reply code/modules/mob/living/silicon/robot/robot_modules.dm]: We as developers decided that It's not necessary to keep cyborg taser module. Please remove it and it's reference replace with regular taser, with it's power source with infinite power cell.

**Comment by Arrow768** [comment]
While I am not necessarily opposed to the removal of the sec borgs, I believe that a replacement for them should be added at the same time.

File: html/changelogs/paradoxspace-notthelaw.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscdel: "Due to various human rights abuses and poor quality programming, NanoTrasen has removed Security programming from their stationbound units."
```
Comment (Arrow768) [inline html/changelogs/paradoxspace-notthelaw.yml]: That is something for a IC changelog and not for the ooc changelog.
The OOC changelog should contain the changes stated plainly without giving it a IC spin.

**Comment by ParadoxSpace** [comment]
A replacement won't be coming with this PR.

File: html/changelogs/paradoxspace-notthelaw.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscdel: "The Security cyborg module has been removed."
```
Comment (Arrow768) [inline html/changelogs/paradoxspace-notthelaw.yml]:
```suggestion
  - rscdel: "The security cyborg module has been removed."
```

**Comment by ParadoxSpace** [comment]
Why?

**Comment by skull132** [comment]
To be merged and reviewed this week.

**Comment by Karolis2011** [comment]
Please also remove cyborg taser module. And it's use in Bluespace Borg.

**Comment by Kaedwuff** [comment]
There was a heavy push in the thread to replace the security borg with something new.

The last time I tried to remove a major feature that nobody liked in the game without providing something new that PR was not accepted. I would like this PR to be held to the same standards.

**Comment by skull132** [comment]
> The last time I (and others) tried to remove a major feature that nobody liked in the game without providing something new that PR was not accepted. I would like this PR to be held to the same standards.

That's. Not how development works. There can never be a standard of, "We aren't removing things, we are going to rework them instead." There is no way to rework sec borgs without making them not be sec borgs, since the major issues are largely influenced by them being borgs.

**Comment by OneOneThreeEight** [comment]
Likewise, keep in mind methodologies such as, "Rework instead of remove" were already tried and absolutely 0 people had better ideas on what to do with security borgs. "Rework instead of remove" is a cool unofficial policy, but it cannot be an absolute one to follow. Otherwise we'd be at this issue forever.

**Comment by NonQueueingMatt** [comment]
GAMER DOWN GAMER DOWN

**Pull request merged**